webapp/portlet/src/main/webapp/vue-apps/favorites-list-top-bar/components/ActivityFavoriteItem.vue

```diff
@@ -1,14 +1,20 @@
 <template>
   <v-list-item class="clickable" :href="activityUrl">
-    <v-list-item-icon class="me-3 my-auto">
-      <v-icon size="22" class="icon-default-color"> fas fa-file-alt </v-icon>
+    <v-list-item-icon v-if="activityTypeExtension && activityTypeExtension.img" class="me-3 my-auto">
+      <v-img
+        :src="activityTypeExtension.img"
+        max-height="28"
+        max-width="25" />
+    </v-list-item-icon>
+    <v-list-item-icon v-else class="me-3 my-auto">
+      <v-icon size="24" :class="activityTypeExtension.class"> {{ activityTypeExtension.icon }} </v-icon> 
     </v-list-item-icon>
 
     <v-list-item-content>
       <v-list-item-title class="text-color body-2">
         <p
           class="ma-auto"
-          v-sanitized-html="activityTitle"></p>
+          v-sanitized-html="activityTypeExtension.title"></p>
```

Comment:
```suggestion
          v-sanitized-html="activityTitle"></p>
```

and add this computed value:

```javascript
activityTitle() {
  return this.activityTypeExtension?.title || this.activity.title;
}
```

Reply:
done